docs/3.0-migration-guide.d/remove_MBEDTLS_X509_CHECK_x_KEY_USAGE_options.md

```diff
@@ -0,0 +1,20 @@
+Remove `MBEDTLS_X509_CHECK_*_KEY_USAGE` options from `config.h`
+--
+
+This change affects users who have chosen the compilation time options to disable
+the library's verification of the `keyUsage` and `extendedKeyUsage` fields of an x509
+certificate.
+
```

Comment:
Instead of the two following paragraphs, I propose:
"The MBEDTLS_X509_CHECK_KEY_USAGE and MBEDTLS_X509_CHECK_EXTENDED_KEY_USAGE configuration options are removed and the X509 code now behave as if they were always enabled. It is consequently not possible anymore to disable at compile time the verification of the `keyUsage` and `extendedKeyUsage` fields of X509 certificates."